(WIP) Check pickling of ufuncs - reopened

sdegryze opened the pull request:
reopened in an attempt to fix triggering travis build

ogrisel commented on setup.py:
```diff
@@ -12,7 +12,9 @@
+    'mock',
+    'numpy',
+    'scipy'
 ]
```
The `test_requirements` have been dropped from the master branch in favor of a `dev-requirements.txt` file. As installing numpy / scipy is currently very costly under Linux (built from source when installed with pip or setuptools) I would rather not put them as a dependency in `dev-requirements.txt`) but instead upgrade the travis config to use miniconda, at least on some of the entries of the build matrix and only put those dependencies in the `.travis.yml` file.

ogrisel commented:
@sdegryze if you want I can help do the squashing & rebasing of this PR.

sdegryze commented:
Thanks @ogrisel - it took me a while but it looks like Travis is happy now. I have no clue why builds were not triggered before. I rebased, squashed the commits, and opened a new PR (https://github.com/cloudpipe/cloudpickle/pull/34). Still learning how to go through these steps - appreciate your patience :)

sdegryze commented:
Closing this PR since a squashed version is opened under #34

ogrisel commented:
Note for later: you can `git push --force` the rebased / squashed version to the same branch to get the same pull request updated. No need to create a new one.
